saleor/payment/migrations/0033_payment_complete_order.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.5 on 2021-08-30 08:09
```

Comment:
Can we merge these migrations into one?

Reply:
Let's ignore and deal with it when merging `feature/partial-payments`. I'm expecting a dozen of conflicts with `3.0` / `master` which will have to be resolved anyway.

Reply:
👍